Documentation/known-issues.md

```diff
@@ -0,0 +1,61 @@
+Known issues & workarounds
+==========================
+
+This document outlines the known issues and workarounds for the current state of 
+the CLI tools. Issues will also have a workaround and affects sections if necessary. You can use this page to 
+get information and get unblocked. 
+
+This page lists only the major issues that block users in doing their everyday tasks and that affect all or 
+most of the commands in the CLI tools. For minor issues, your best bet is still to check out the issues in 
+this repo. 
+
+# Current issues (12/23/2015)
+
+## El Capitan support
+El Capitan's RID is not supported properly currently. 
+
+**Issues tracking this:** 
+
+* [#498](https://github.com/dotnet/cli/issues/498)
+* [#291](https://github.com/dotnet/cli/issues/291)
+
+**Affects:** most of the commands, but more than likely you will not be able to 
+use `dotnet compile` and `dotnet-run` on El Capitan. For others, there is a 
+workaround.
+
+**Workaround:** use the --runtime switch with the value of `osx.10.11-x64` in 
+`dotnet restore` and `dotnet publish` and you will be able to run your app from 
+the published directory. 
+
+## Resolving the Standard library packages
+The StdLib package is on a MyGet feed. In order to restore it, a MyGet feed needs to be added 
```

Comment:
@brthor I know, I left it in to guide people to update and use `dotnet new`from the new vits.